src/main/java/com/github/speedwing/log4j/cloudwatch/appender/CloudwatchAppender.java

```diff
@@ -85,54 +90,60 @@ public void setMessagesBatchSize(int messagesBatchSize) {
 
     @Override
     protected void append(LoggingEvent event) {
+        // Make sure that we don't get stuck in an infinite loop
+        if (event.getLocationInformation().getClassName().startsWith("com.amazonaws")) {
```

Comment:
Not sure about this? HAve you checked if `Filter`s coudl be used? I would not hardcode this if possible, but recur to this only as last chance if we don't find a better solution.